src/Components/Voting/ArticleVotes.js

```diff
@@ -0,0 +1,79 @@
+import { patchArticleVote } from "../../Api/Api";
+import { useState, useEffect } from "react";
+import arrowDown from "../../Assets/arrow-down.svg";
+import arrowUp from "../../Assets/arrow-up.svg";
+import arrowDownOrange from "../../Assets/arrow-down-orange.svg";
+import arrowUpOrange from "../../Assets/arrow-up-orange.svg";
+
+const ArticleVotes = ({ article, articleId }) => {
+  const [votes, setVotes] = useState(article.votes);
+  const [hasUpVoted, setHasUpVoted] = useState(false);
+  const [hasDownVoted, setHasDownVoted] = useState(false);
+
+  const patchVote = async (value) => {
+    try {
+      const response = await patchArticleVote(articleId, value);
+    } catch (error) {
+      console.log(error);
+      setVotes(votes + value);
```

Comment:
Bear in mind that if you rely on the current state to determine the new one, you should use a callback function - you may drop lucky otherwise
```js
setVotes((currentVotes) => currentVotes + value)
```